cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/Input/TextInputWithError.tsx

```diff
@@ -0,0 +1,44 @@
+// React imports
+import React, { ChangeEvent } from 'react';
+
+// Library imports
+import { CdsInput } from '@cds/react/input';
+import { useFormContext } from 'react-hook-form';
+import { CdsControlMessage } from '@cds/react/forms';
+
+interface Props {
+    className?: string;
+    defaultValue?: string;
+    label: string;
+    name: string;
+    placeholder?: string;
+    handleInputChange: (field: any, value: string) => void;
+    textHidden?: boolean;
+}
+function TextInputWithError(props: Props) {
+    const { className, defaultValue, label, name, handleInputChange, placeholder, textHidden } = props;
```

Comment:
nit: suggest `maskText` instead of `textHidden`

Reply:
renamed